src/App/Fossa/Container/AnalyzeNative.hs

```diff
@@ -13,6 +13,8 @@ import App.Fossa.Config.Common (
  )
 import App.Fossa.Config.Container.Analyze (
   ContainerAnalyzeConfig (arch, dockerHost, filterSet, imageLocator, onlySystemDeps, revisionOverride, scanDestination),
+  JsonOutput (JsonOutput),
+  jsonOutput,
```

Comment:
`jsonOutput` can go in the `ContainerAnalyzeConfig` import, like this:

```haskell
import App.Fossa.Config.Container.Analyze (
  ContainerAnalyzeConfig (arch, dockerHost, filterSet, imageLocator, jsonOutput, onlySystemDeps, revisionOverride, scanDestination),
  JsonOutput (JsonOutput),
 )
```

But that list is getting long and we've imported pretty much everything from `ContainerAnalyzeConfig`, so it might be even better to just do this:

```haskell
import App.Fossa.Config.Container.Analyze (
  ContainerAnalyzeConfig (..),
  JsonOutput (JsonOutput),
 )
```